platformio.ini

```diff
@@ -14,7 +14,7 @@ board = ttgo-lora32-v1
 build_flags =
 	${env.build_flags}
 	-DMONITOR_CLASS=DisplayMonitor
-	-DBLINK_LED=2
+	-DBLINK_LED=14
```

Comment:
This should not be changed, since ttgo-lora32-v1 LED is in pin 2.

Reply:
Change made!